modules/bird2.nix

```diff
@@ -19,6 +19,7 @@ in
 
     services.bird2 = {
       enable = true;
+      checkConfig = lib.mkIf cfg.roagen.enable false;
```

Comment:
Maybe a stub for check time would be helpful. NixOS provides the option to execute commands before the config check is executed: https://search.nüschtos.de/?query=bird2&option_scope=11&option=services.bird2.preCheckConfig

Reply:
Yup, makes sense. Fixed.